apis/content/contentapi.yaml

```diff
@@ -1655,7 +1661,113 @@ paths:
         CollectParameters: false
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
-        IsMultiContentStreaming: false      
+        IsMultiContentStreaming: false
+
+
+
+  /collaborator/update/{Content_ID}:
+     patch:
+      description: >-
+        This API is associated with adding or removing collaborators in a content on the Sunbird Platform .
+          - **_/collaborator/update/{Content_ID}_** endpoint executes the "Update Collaborators" request based on parameters provided as metadata in the request body.
+          - {...}refers to a section of endpoint URL path that needs to be replaced by appropriate Id.
+          - It is a must to provide values for the required parameters
+      summary: Update Collaborators
+      tags:
+        - Content APIs
+      operationId: Update Collaborators
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: Content_ID
+          type: string
+          required: true
+          description: 'Please append a valid Content ID To the Request URL'   
+        - name: Body
+          in: body
+          required: true
+          description: >- 
+            The body parameter refers to the format of the request, the request body (payload) takes the following media types to make the payload:'
+            
+            - JSON 
+            - XML data
+          
+            The body is the representation of the resource object for  updating the content.
+          schema:
+            $ref: '#/definitions/UpdateCollaboratorsRequest'
+        - name: Content-Type
+          in: header
+          required: true
+          type: string
+          description: >-
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+              - Multipart/form-data
+              - Application/x-www-form-urlencoded
+        - name: X-Authenticated-User-Token
+          in: header
+          required: true
+          type: string
+          description: 'Represents the authorized User token to execute the API.'
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: >-
+            SUCCESS. The **Update Collaborators** operation was successful!
+          schema:
+            $ref: '#/definitions/ContentCreateResponse'
```

Comment:
This API is not for getting ContentCreateResponse or ContentCreateErrorResponse. If you want to use common definitions then use CommonSuccessResponse and CommonErrorResponse.